media-libs/audiofile/audiofile-0.3.6.recipe

```diff
@@ -30,9 +30,10 @@ Supported compression formats:
 HOMEPAGE="https://audiofile.68k.org/"
 COPYRIGHT="1998-2013 Michael Pruett, Chris Pirazzi, Scott Porter, Doug Scott"
 LICENSE="GNU LGPL v2.1"
-REVISION="4"
+REVISION="5"
 SOURCE_URI="https://audiofile.68k.org/audiofile-0.3.6.tar.gz"
 CHECKSUM_SHA256="cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+PATCHES="audiofile-$portVersion.patchset"
 
 ARCHITECTURES="x86 x86_64"
 SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
```diff
-PATCHES="audiofile-$portVersion.patchset"
+
+SOURCE_FILENAME_2="audiofile-0.3.6-gcc6-build-fixes.patch"
+srcGitRev_2="1709d2d431a493deea7765902011730e04fab466"
+SOURCE_URI_2="https://gitweb.gentoo.org/repo/gentoo.git/plain/media-libs/audiofile/files/$SOURCE_FILENAME_2?id=$srcGitRev_2#noarchive"
+CHECKSUM_SHA256_2="c088e405ae54bc560e7d203cc5a18edb829b555e675c363d789b0f3f2b4d5d0c"
 
-ARCHITECTURES="x86 x86_64"
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

```diff
 
+PATCH()
+{
+	patch -p1 -i "$sourceDir2"/$SOURCE_FILENAME_2
+}
+
 BUILD()
```

It works like a charm.
Since I've already edited the recipe, I can update your PR :)